src/winetricks

```diff
@@ -2637,13 +2646,15 @@ w_metadata()
 
     # Do sanity check unless running on Cygwin, where it's way too slow.
     case "$W_PLATFORM" in
-        windows_cmd)
-            ;;
-        *)
+        windows_cygwin) w_warn "FIXME: Sanity check for w_metadata() is too slow, skipping.." ;;
+        windows_windows) w_die "FIXME: Platform $W_PLATFORM is not implemtented in w_metadata()" ;;
+        nix_redox) w_die "FIXME: Platform $W_PLATFORM is not implemtented in w_metadata()" ;;
```

Comment:
Not needed.

Reply:
@austin987 `nix_redox` is not tested, do you want me to assume that it works?

Reply:
No, I don't want extra platforms added for no reason, particularly if they're untested. There are literally hundreds of possibilities, there's absolutely no reason to detect all of them unless they need special treatment somewhere.